app/routers/sessions.py

```diff
@@ -29,7 +31,9 @@ async def create_session(session: Session):
         # since we know that there is going to be only one question set for now
         if "question_sets" in quiz and quiz["question_sets"]:
             question_set_id = quiz["question_sets"][0]["_id"]
-            questions = client.quiz.questions.find({"question_set_id": question_set_id})
+            questions = client.quiz.questions.find(
+                {"question_set_id": question_set_id}, sort=[("_id", pymongo.ASCENDING)]
+            )
```

Comment:
why do we need questions in ascending order?

Reply:
Why sort by `id`? If we switch to a random UUID (isn't it already random), this will cause a problem, no? Questions should have an ordering field in the quiz, I think

Reply:
in the session object, we are storing the first 10 session answers right? (the subset pattern)
so, the session answers have to be stored in the order of the questions in the quiz. If I don't sort it, the questions appear in some random order.

@pritamps the ID generated is already a UUID - only the name of the field is ID. Example image added!
<img width="315" alt="Screenshot 2022-04-27 at 10 30 04 AM" src="https://user-images.githubusercontent.com/13185699/165443978-ed39957f-bd5e-4751-b4ad-fab7b6044783.png">

Reply:
I think i have some basic misunderstanding of what's going on. Then how does sorting by ID work if it's supposed to be random?

Reply:
In my answer above, I didn't mention that the ID was random right? It is auto-incremented, but it is unique enough to do the job of the UUID. The random part that I did mention above was the randomness in the order of the questions returned if we do not sort!

Reply:
Aiyo... The purpose of using UUID is lost if you auto-increment them. A user with access to one UUID will then have access to all subsequent UUIDs in the session. So in this case, there's no real advantage to using UUID at all :-/